guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,7 +1,12 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
-When you run a remote job on hosts, for every host, {Project} performs the following actions to find a remote execution {SmartProxy} to use.
+Remote execution can be performed on the hosts that are registered through a {SmartProxy}.
```

Comment:
You can also perform REX directly through `{ProjectServer}`.

Reply:
I'm not 100% sure about what those curly placeholders get filled with, but REX jobs always go through a smart proxy, even though it may be colocated on the same machine

Reply:
> I'm not 100% sure about what those curly placeholders get filled with.

Curly brackets will replace the Project name as those are attributes. :slightly_smiling_face: 

> but REX jobs always go through a smart proxy, even though it may be colocated on the same machine

@maximiliankolb I am confused :smile: +

Reply:
I guess what I'm trying to say: You can also attach a host to Foreman; so REX works through Foreman's internal Smart Proxy.

Foreman would be your ProjectServer, but not a "SmartProxyServer".

```suggestion
Remote execution can be performed on managed hosts through {SmartProxies}.
```

Reply:
Addressed :+1: